drivers/staging/media/sunxi/cedar/ve/Kconfig

```diff
@@ -0,0 +1,8 @@
+config VIDEO_SUNXI_CEDAR_VE
+	tristate "Allwinner CedarX VideoEngine Driver"
```

Comment:
That will be interesting to test also someone was wishing for shared buffer option for working video Overlay engine.